packages/office-ui-fabric-react/src/components/Conformance.test.tsx

```diff
@@ -0,0 +1,212 @@
+import * as React from 'react';
+import * as renderer from 'react-test-renderer';
+
```

Comment:
While you have a description in your GitHub PR, it would be good to add a readme-like blob here describing what exactly does "conformance" look like (aka describe in words and provide examples of what this test tests for).